### ISSUE TITLE
Fix link for `class` attribute specification in error message of `invalid-attr`

### ISSUE THREAD

**Issue by kagankan** [markuplint/markuplint#565]
- Markuplint version: `2.11.0`
- Parser lang: HTML 
- Node.js version: `16.16.0`
- OS: Windows

## Describe the bug

Related to #564,
markuplint shows the URL of `SVG` specification for HTML class error

```
error: 属性「class」の値の部分はCSS構文として「<ident-token>*」である必要があります (https://www.w3.org/TR/SVG/styling.html#ClassAttribute) (invalid-attr)
```

## Code Example

```html
<div class="md:flex">
...
</div>
```

## Steps To Reproduce

Lint HTML file including the `Code Example` with `markuplint:recommended` config

## The current behavior

shows `SVG` specification

## The expected behavior

shows `HTML` specification
https://html.spec.whatwg.org/multipage/dom.html#classes
